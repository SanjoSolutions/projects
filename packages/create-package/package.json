{
  "name": "@sanjo/create-package",
  "version": "0.2.0",
  "type": "module",
  "description": "Create package",
  "keywords": [],
  "homepage": "https://github.com/SanjoSolutions/projects/tree/main/packages/create-package",
  "license": "Unlicense",
  "main": "./build/index.js",
  "types": "./build/index.d.ts",
  "exports": {
    ".": {
      "types": "./build/index.d.ts",
      "default": "./build/index.js"
    },
    "./*.js": {
      "types": "./build/*.d.ts",
      "default": "./build/*.js"
    }
  },
  "bin": "./build/bin/create-package.js",
  "repository": {
    "type": "git",
    "url": "https://github.com/SanjoSolutions/projects.git",
    "directory": "packages/create-package"
  },
  "dependencies": {
    "@sanjo/escape-for-reg-exp": "^1.0.0",
    "@sanjo/package-name-to-folder-name": "^1.0.0",
    "@sanjo/read-file": "^2.0.0",
    "@sanjo/write-file": "^1.0.0",
    "lodash.sortby": "^4.7.0",
    "lodash.sorteduniqby": "^4.7.0",
    "ncp": "^2.0.0"
  },
  "devDependencies": {
    "@sanjo/diff-folders": "^1.0.0",
    "@sanjo/exec": "^1.0.0",
    "@sanjo/fs": "^1.0.0",
    "@sanjo/make-temporary-directory": "^1.0.0",
    "@sanjo/test-create-package": "^0.2.0",
    "@types/lodash.sortby": "^4.7.6",
    "@types/lodash.sorteduniqby": "^4.7.6",
    "@types/ncp": "^2.0.4"
  },
  "publishConfig": {
    "access": "public"
  }
}
