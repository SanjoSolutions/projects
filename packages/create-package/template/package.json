{
  "name": "<PACKAGE_NAME>",
  "version": "0.0.0",
  "type": "module",
  "description": "<PACKAGE_DESCRIPTION>",
  "keywords": [],
  "homepage": "https://github.com/SanjoSolutions/unnamed/tree/main/packages/<PACKAGE_FOLDER_NAME>",
  "license": "Unlicense",
  "main": "./index.js",
  "repository": {
    "type": "git",
    "url": "https://github.com/SanjoSolutions/unnamed.git",
    "directory": "packages/<PACKAGE_FOLDER_NAME>"
  },
  "dependencies": {},
  "devDependencies": {},
  "types": "./index.d.ts",
  "publishConfig": {
    "access": "public"
  }
}
