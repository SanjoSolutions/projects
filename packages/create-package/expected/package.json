{
  "name": "@sanjo/test-package-for-create-package",
  "version": "0.0.0",
  "type": "module",
  "description": "Description of test package",
  "keywords": [],
  "homepage": "https://github.com/SanjoSolutions/unnamed/tree/main/packages/test-package-for-create-package",
  "license": "Unlicense",
  "main": "./index.js",
  "repository": {
    "type": "git",
    "url": "https://github.com/SanjoSolutions/unnamed.git",
    "directory": "packages/test-package-for-create-package"
  },
  "dependencies": {},
  "devDependencies": {},
  "types": "./index.d.ts",
  "publishConfig": {
    "access": "public"
  }
}
